teemaderegister-react/src/components/SettingsAccount.jsx

```diff
@@ -29,6 +29,7 @@ const propTypes = {
       profile: shape({
```

Comment:
frontend wise double check linter, seems like not all changes are correct in the files. Maybe I am wrong.